### PR TITLE
PHP windows setup

### LINKED ISSUE #2180: PHP windows_setup
### Description

Documentation on how to install php in windows OS.

### Domain

Backend Dev (PHP)

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by vasu-1**
<hr>

## Description 📜

Setting up PHP in Windows using xampp.
Fixes #2180 

<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Backend Dev #(PHP)

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

File: Backend_Development_With_PHP/Getting_Started/Windows_Setup/readme.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+1. Open xampp from explorer menu and start apache server.
+
+![xampp.png](xampp.PNG)
+
+2. Start Apache Server
+3. Now open your favourite browser and type ` localhost ` in the searchbar.
+4. if you see this following output then Congratulations !! you have successfully installed PHP on your machine.
+
+![localhost.png](localhost.PNG)
+<br/>
```
Comment (praveenscience) [inline Backend_Development_With_PHP/Getting_Started/Windows_Setup/readme.md:34]: Get rid of `<br />`.

File: Backend_Development_With_PHP/Getting_Started/Windows_Setup/readme.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+
+## Contributor
+[![vasu-1](https://contrib.rocks/image?repo=vasu-1/vasu-1)](https://github.com/vasu-1)
```
Comment (praveenscience) [inline Backend_Development_With_PHP/Getting_Started/Windows_Setup/readme.md:37]: No need of this na?

File: Backend_Development_With_PHP/Getting_Started/readme.md [hunk 3]
```diff
@@ -1 +1,3 @@
-### GETTING STARTED
+## GETTING STARTED
+
+[Windows_Setup](Windows_Setup)
```
Comment (praveenscience) [inline Backend_Development_With_PHP/Getting_Started/readme.md:3]: Why underscore in text?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Reviewed.

File: Backend_Development_With_PHP/Getting_Started/Windows_Setup/readme.md (hunk 2 above)
Comment (vasu-1) [reply Backend_Development_With_PHP/Getting_Started/Windows_Setup/readme.md:37]: I saw this many contributors are writing this so . i have added this. if you want you can remove it !! Np

**Comment by vasu-1** [comment]
if no changes are required then please merge this pr @praveenscience !!

**Review by praveenscience — APPROVED** [review]
Looks good to me! 👍

**Comment by meinhoonharsh** [comment]
It's good to go, Fine from my side too

**Comment by vasu-1** [comment]
> It's good to go, Fine from my side too

If it is fine then approve it !!!!!

**Review by meinhoonharsh — APPROVED** [review]

**Review by meinhoonharsh — APPROVED** [review]

**Pull request merged**